modules/serverless-installing-cli-linux-ibm-z_.adoc

```diff
@@ -0,0 +1,9 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="installing-cli-linux_ibm_z_{context}"]
+= Installing the `kn` CLI for Linux on IBM Z and LinuxONE
+
+include::serverless-installing-cli-linux-ibm-z_tarball.adoc[leveloffset=+1]
+include::serverless-installing-cli-linux-ibm-z_rpm.adoc[leveloffset=+1]
```

Comment:
You can't include modules in a module afaik? I could be wrong but I think modules need to be included in an assembly?